### ISSUE TITLE
Time field blinking

### ISSUE THREAD

**Issue by timparenti** [concerto/concerto#1451]
At the current state of `master`, the time field blinks somewhat randomly in Firefox 52.2.1 ESR.

* **Expected behavior:** Every 15 seconds, the current time is re-evaluated and the contents of the time field are seamlessly replaced.
* **Observed behavior:** Every 15 seconds, the contents of the time field disappear entirely, only to reappear at a later time.  Sometimes it reappears a mere fraction of a second later, but it usually takes several seconds, and sometimes does not reappear within any given 15-second period.  The accuracy of the time, when it is displayed, does not appear to be affected.

Other text fields do not appear to exhibit the same blinking behavior, even when they only contain one item.  Checking the console, there do not appear to be any content updates coming through for the time field.

In my testing, I have observed that this behavior does not occur in Firefox 45.3.0 ESR, Firefox 48.0, or Chrome 58.0.3029.110.  This came up in testing a new upgrade from Concerto 2.3.1, so perhaps some settings have changed that I am missing?

**Comment by mfrederickson** [comment]
I wonder if a field setting/config for entry and exit transitions of "replace" would prevent the flicker?

**Comment by timparenti** [comment]
It does not appear that that helped.

Upon further observation, it seems that after the content disappears, it only reappears at such time as a different transition begins elsewhere on the screen.  (Since our primary graphics content has durations both above and below 15 seconds, this explains the somewhat random aspect of this behavior.)  The animations inspector in Firefox does show that it is, however, attempting to execute a pair of animations every 15 seconds (presumably an exit and an entry).  The content just doesn't always show up.

Looking closer, even our text fields with explicit "replace" transitions exhibit this flicker according to the same pattern.  I didn't notice it earlier because ours uses a longer content duration and is typically fairly synchronized with another element, so the effect was less pronounced.

**Comment by timparenti** [comment]
Anything more I can be checking on the client-side to help pin down the source of this problem?

**Comment by mfrederickson** [comment]
I don't think so, but if you'd like to dig into the concerto/concerto-frontend repo and try to figure it out, that would be great.

**Comment by whippc** [comment]
I can confirm this issue still occurs in the lastest versions of Firefox esr and luakit on the latest raspian.

**Comment by mfrederickson** [comment]
The frontend uses the polymer neon-animation class which now appears to be deprecated.  https://www.webcomponents.org/element/PolymerElements/neon-animation

I can confirm that FF is still showing the problem as of today, and Chrome does not.

Looks like we need to overhaul the frontend code at some point.
